aws/routes/routeAwsStatus.go

```diff
@@ -30,12 +30,15 @@ func getAwsAccountsStatus(r *http.Request, a routes.Arguments) (int, interface{}
 		l.Error("failed to get AWS accounts' bill repositories", err.Error())
 		return 500, errors.New("failed to retrieve bill repositories")
 	}
+	// Code unneeded for now
```

Comment:
Can you explain why this code is no more needed, please?

Reply:
Made the comment clearer.

Btw do tell if you think the code should simply be removed, I kept it in the case it was there to be used in some later functionality.

Reply:
It is better to explain directly in the code why this is not necessary at the moment. We will keep it, however. You never know.